src/internal/tester/config.go

```diff
@@ -112,36 +112,31 @@ func readTestConfig() (map[string]string, error) {
 		TestCfgUserID,
 		os.Getenv(EnvCorsoM365TestUserID),
 		vpr.GetString(TestCfgUserID),
-		"LynneR@10rqc2.onmicrosoft.com",
-	)
+		"LynneR@10rqc2.onmicrosoft.com")
 	fallbackTo(
 		testEnv,
 		TestCfgSecondaryUserID,
 		os.Getenv(EnvCorsoSecondaryM365TestUserID),
 		vpr.GetString(TestCfgSecondaryUserID),
-		"AdeleV@10rqc2.onmicrosoft.com",
-	)
+		"AdeleV@10rqc2.onmicrosoft.com")
 	fallbackTo(
 		testEnv,
 		TestCfgLoadTestUserID,
 		os.Getenv(EnvCorsoM365LoadTestUserID),
 		vpr.GetString(TestCfgLoadTestUserID),
-		"leeg@10rqc2.onmicrosoft.com",
-	)
+		"leeg@10rqc2.onmicrosoft.com")
 	fallbackTo(
 		testEnv,
 		TestCfgLoadTestOrgUsers,
 		os.Getenv(EnvCorsoM365LoadTestOrgUsers),
 		vpr.GetString(TestCfgLoadTestOrgUsers),
-		"AdeleV@10rqc2.onmicrosoft.com,LynneR@10rqc2.onmicrosoft.com",
-	)
+		"AdeleV@10rqc2.onmicrosoft.com,LynneR@10rqc2.onmicrosoft.com")
 	fallbackTo(
 		testEnv,
 		TestCfgSiteID,
 		os.Getenv(EnvCorsoM365TestSiteID),
 		vpr.GetString(TestCfgSiteID),
-		"10rqc2.sharepoint.com,bb5d5b4a-e089-4e66-9868-9e263ecc635d,4fa3a2c0-fa81-4e6f-8e8b-1479a8927bc6",
-	)
+		"10rqc2.sharepoint.com,4892edf5-2ebf-46be-a6e5-a40b2cbf1c1a,38ab6d06-fc82-4417-af93-22d8733c22be")
```

Comment:
is the purge script setup to handle this SharePoint site?

Reply:
The purge script runs on whatever secret var we populate in our GH env.  I've set up the current site with the same environment expectations (two drives, test folder, one item each), so if the script is running on the current site successfully, we _should_ be able to swap between the two without issue.